nautobot/dcim/models/racks.py

```diff
@@ -506,7 +506,8 @@ def get_utilization(self):
                 available_units.remove(u)
 
         occupied_unit_count = self.u_height - len(available_units)
-        percentage = int(float(occupied_unit_count) / self.u_height * 100)
+        # Return the numerator and denominator as percentage is to be calculated later where needed
+        return (occupied_unit_count, self.u_height)
```

Comment:
Anywhere you return a tuple, please return a `collections.namedtuple` instead.

See: https://gist.github.com/lucasnad27/e93b8f316259be3299e5#clarify-multiple-return-values-with-named-tuples